version: '2'
services:
  bitcoind:
    environment:
      BITCOIN_EXTRA_ARGS: |
        rpcauth=eps:7a32cf23721f62944b9df273fb21f869$$39882086c2df8e9f91dcdbc34ec9bb515f6ee66cd31eefdab7268fe4f8945c3f
  electrum_ps:
    image: docker.io/peleion/eps
    restart: unless-stopped
    ports:
      - 50002:50002
    volumes:
      - "electrum_ps_datadir:/data"
    environment:
      - DAEMON_URL=eps:cmhKyeLSco35KVhecK3blYcHFx73xrJ1fOQtjtmiKks=@bitcoind:43782
volumes:
  electrum_ps_datadir:
incompatible:
  - pruning
